VERSIONS.md

```diff
@@ -1,32 +1,32 @@
-| Version | iOS version | Android version | Common version | BillingClient version | Unity IAP compatibility  |
```

Comment:
I removed the `BillingClient version` column here... I think it's not that useful and helps simplify things a bit.

Reply:
It's _somewhat_ useful for folks using observer mode, but the vast majority of observer mode devs in Unity will be using Unity IAP, so as long as we still have that one this seems okay.

Reply:
I agree with @aboedo 

I don't think anyone is using BillingClient directly when in observer mode